docs/api/structures/web-contents-stream-result.md

```diff
@@ -0,0 +1,6 @@
+# WebContentsStreamResult Object
+
+* `id` String - The identifier of a WebContents stream, this identifier can be
+  used with [`navigator.webkitGetUserMedia`]. The identifier may be empty if
+  not requested from a renderer process.
+* `mediaId` String - The DesktopMediaID associated with the WebContents.
```

Comment:
When might a consumer need to know what the `mediaId` is?

Reply:
I exposed it thinking it may be useful for debugging. I don't think it has any other uses at the moment.
`id` looks like `jzR0r6ua6Ixl5//h8NZv5w==`, `mediaId` looks like `web-contents-media-stream://4:3`